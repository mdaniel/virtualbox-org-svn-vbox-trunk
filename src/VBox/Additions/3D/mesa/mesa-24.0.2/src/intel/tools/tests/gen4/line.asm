line(16)        null<1>F        g3.4<0,1,0>F    g6<8,8,1>F      { align1 compr };
line(8)         null<1>F        g4.4<0,1,0>F    g37<8,8,1>F     { align1 };
