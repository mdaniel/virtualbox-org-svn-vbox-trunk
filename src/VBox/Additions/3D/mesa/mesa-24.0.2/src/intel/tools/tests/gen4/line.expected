59 20 80 00 bc 77 00 20 70 00 00 00 c0 00 8d 00
59 00 60 00 bc 77 00 20 90 00 00 00 a0 04 8d 00
